placeholders.go

```diff
@@ -26,9 +26,9 @@ func parsePlaceholders(input string, r *http.Request) (string, error) {
 		case "{fragment}":
 			input = strings.Replace(input, "{fragment}", r.URL.Fragment, -1)
 		case "{host}":
-			input = strings.Replace(input, "{host}", r.URL.Host, -1)
+			input = strings.Replace(input, "{host}", r.Host, -1)
 		case "{hostonly}":
-			input = strings.Replace(input, "{hostonly}", r.URL.Hostname(), -1)
+			input = strings.Replace(input, "{hostonly}", r.Host, -1)
```

Comment:
This one needs the port cleanup code as we take the plain r.Host, which still can have a port.